instrumentation/tests/java/test.sh

```diff
@@ -25,9 +25,9 @@ echo "$OUTPUT" | grep "SPLUNK_METRICS_ENABLED=abcd" > /dev/null && echo "Test pa
 echo "Test presence of ANOTHER_VAR"
 echo "$OUTPUT" | grep "ANOTHER_VAR=foo" > /dev/null && echo "Test passes"  || exit 1
 echo "Test absence of FOO"
-echo "$OUTPUT" | grep -v "FOO" > /dev/null && echo "Test passes"  || exit 1
+echo "$OUTPUT" | grep "FOO" | grep -v "FOO" > /dev/null && echo "Test passes"  || exit 1
```

Comment:
Maybe simpler to do something like:
```
if echo "$OUTPUT" | grep -q "FOO"; then
  exit 1
else
  echo "Test passes"
fi

Reply:
how about something like
```[[ $OUTPUT != *FOO* ]] && echo "no FOO" || echo "found FOO"```

Reply:
sorry just seeing this. I am not familiar with `-q`, looking.

Reply:
sorry, the regexp approach doesn't work for multiline variables.